core/kernel/asan.c

```diff
@@ -411,11 +411,18 @@ void __asan_handle_no_return(void)
 void __asan_register_globals(struct asan_global *globals, size_t size);
 void __asan_register_globals(struct asan_global *globals, size_t size)
 {
-	size_t n;
+	size_t n = 0;
```

Comment:
The purpose of this initialization is to make sure that whatever mistake we make, at least it has a well-defined value.

I prefer `for (n = 0;...)` below since that's what we had before.

Reply:
Done